docs/sources/alerting/fundamentals/notifications/templates.md

```diff
@@ -19,146 +19,167 @@ labels:
 
-Both labels and annotations have the same structure: a set of named values; however their intended uses are different. The purpose of annotations is to add additional information to existing alerts.
+Here’s an example of templating a `severity` label based on the query value:
 
-There are a number of suggested annotations in Grafana such as `description`, `summary`, `runbook_url`, `dashboardUId` and `panelId`. Like labels, annotations must have a name, and their value can contain a combination of text and template code that is evaluated when an alert is fired.
+```
+{{ if (gt $values.A.Value 90.0) -}}
+critical
+{{ else if (gt $values.A.Value 80.0) -}}
+high
+{{ else if (gt $values.A.Value 60.0) -}}
+medium
+{{ else -}}
+low
+{{- end }}
+```
 
-Here is an example of templating an annotation in the context of an alert rule. The text/template is added into the Add annotations section.
+Avoid using query values in labels, as this may result in the creation of numerous alerts when only one is needed. Use annotation to inform about the query value instead.
 
-        CPU usage for {{ index $labels "instance" }} has exceeded 80% for the last 5 minutes
+For more details on how to template labels, refer to [Template annotations and labels](ref:templating-labels-annotations).
 
-The outcome of this template would print
+## Template notifications
 
-        CPU usage for Instance 1 has exceeded 80% for the last 5 minutes
+[Notification templates](ref:notification-messages) allow you to customize the content of your notifications, such as the subject of an email or the body of a Slack message.
 
-### Template notifications
+Notification templates differ from templating annotations and labels in the following ways:
 
-Notification templates represent the alternative approach to templating designed for reusing templates. Notifications are messages to inform users about events or conditions triggered by alerts. You can create reusable notification templates to customize the content and format of alert notifications. Variables, labels, or other context-specific details can be added to the templates to dynamically insert information like metric values.
+- Notification templates are assigned to the **Contact point**, rather than the alert rule.
+- If not specified, the contact point uses a default template that includes relevant alert information.
+- You can create reusable notification templates and reference them in other templates.
+- The same template can be shared across multiple contact points, making it easier to maintain and ensuring consistency.
+- While both annotation/label templates and notification templates use the same templating language, the available variables and functions differ. For more details, refer to the [notification template reference](ref:notification-message-reference) and [annotation/label template reference](ref:templating-labels-annotations).
```

Comment:
Really good point . And we should give it even more visibility. We can do that in a future PR